throughput/spam.go

```diff
@@ -154,13 +152,20 @@ func (s *Spammer) Spam(ctx context.Context, sh SpamHelper, terminate bool, symbo
 	s.tracker.logState(cctx, issuers[0].cli)
 
 	// broadcast transactions
-	s.broadcast(cctx, cancel, sh, accounts, funds, factories, issuers, feePerTx, terminate)
+	err = s.broadcast(cctx, cancel, sh, accounts, factories, issuers, feePerTx, terminate)
+	if err != nil {
+		return err
+	}
```

Comment:
nit: up to you to ignore this comment since this is developer preference - but I prefer using the in-line syntax of `if err := ... ; err != nil {` where it's possible

Reply:
i like the inline syntax, but in this case the line would be too many characters so gonna chose to keep it.
In other cases I agree with this